tools/release/rollback.sh

```diff
@@ -63,3 +64,9 @@ gcloud docker -- push gcr.io/${PROJECT_ID}/datalab-gateway:latest
 echo "Releasing the Datalab image: ${DATALAB_IMAGE}"
 docker tag -f ${DATALAB_IMAGE} gcr.io/${PROJECT_ID}/datalab:local
 gcloud docker -- push gcr.io/${PROJECT_ID}/datalab:local
+
+echo "Pulling the rollback GPU images: ${DATALAB_GPU_IMAGE}"
+gcloud docker -- pull ${DATALAB_GPU_IMAGE}
```

Comment:
Since this is new, we should gracefully handle the situation where the image we are trying to rollback to does not exist.